src/chainstate/burn/operations/mod.rs

```diff
@@ -330,6 +336,35 @@ pub struct PegInOp {
     pub burn_header_hash: BurnchainHeaderHash, // hash of the burn chain block header
 }
 
+#[derive(Debug, PartialEq, Clone, Eq, Serialize, Deserialize)]
+pub struct PegOutRequestOp {
+    pub amount: u64,                 // sBTC amount to peg out, in satoshis
+    pub recipient: PoxAddress,       // Address to receive the BTC when the request is fulfilled
+    pub signature: MessageSignature, // Signature from sBTC owner as per SIP-021
+    pub memo: Vec<u8>,               // extra unused byte
+
+    // common to all transactions
+    pub txid: Txid,                            // transaction ID
+    pub vtxindex: u32,                         // index in the block where this tx occurs
+    pub block_height: u64,                     // block height at which this tx occurs
+    pub burn_header_hash: BurnchainHeaderHash, // hash of the burn chain block header
+}
+
+#[derive(Debug, PartialEq, Clone, Eq, Serialize, Deserialize)]
+pub struct PegOutFulfillOp {
+    pub chain_tip: StacksBlockId, // The Stacks chain tip whose state view was used to validate the peg-out request
+
+    pub amount: u64,           // Transferred BTC amount, in satoshis
+    pub recipient: PoxAddress, // Address to receive the BTC
+    pub memo: Vec<u8>,         // extra unused byte
```

Comment:
Because we're going to have the peg-out fulfillment op consume a UTXO produced by the peg-out request, please update this struct to include a pointer to the peg-out request.  This would have type `(Txid, u32)` to encode the `vout` entry in the Bitcoin transaction.  We use a similar tactic for `PreStx`, `TransferStx`, `StackStx`, and `DelegateStx` -- the latter three operations consume a UTXO produced by `PreStx`.

Reply:
How would this work for multiple fulfillment transactions given a single peg out request? Only the first fulfillment would be able to consume the UTXO.

Reply:
From yesterdays sBTC discussion we concluded that we're initially only going to support a 1:1 relation between fulfillment transactions and peg out requests.

Reply:
Since the UTXO must be the third output of the peg-out request, I'm asserting that `vout` must be 2 and only storing the `Txid` in the `PegOutFulfillOp`.